STAT302 - assignment 2 Q 2.18

In [1]:
#Q 2.18 a
library(Stat2Data)
data(Sparrows)
str(Sparrows)

'data.frame':	116 obs. of  3 variables:
 $ Treatment : Factor w/ 3 levels "control","enlarged",..: 1 1 1 1 1 1 1 1 1 1 ...
 $ Weight    : num  14.9 15 14.3 17 16 16.2 12.4 14.6 12.2 13 ...
 $ WingLength: num  29 31 25 29 30 31.5 23.5 26 25.5 27 ...


In [2]:
model = lm(Weight ~ WingLength, data = Sparrows)
summary(model)


Call:
lm(formula = Weight ~ WingLength, data = Sparrows)

Residuals:
    Min      1Q  Median      3Q     Max 
-3.5440 -0.9935  0.0809  1.0559  3.4168 

Coefficients:
            Estimate Std. Error t value Pr(>|t|)    
(Intercept)  1.36549    0.95731   1.426    0.156    
WingLength   0.46740    0.03472  13.463   <2e-16 ***
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 1.4 on 114 degrees of freedom
Multiple R-squared:  0.6139,	Adjusted R-squared:  0.6105 
F-statistic: 181.3 on 1 and 114 DF,  p-value: < 2.2e-16


H0: beta1 = 0, Ha: beta1 not 0 \
t = 13.463 df = 114 \
P-value < 2e-16 \
Reject H0 \
We have evidence that winglength can effect the sparrows's weight.

In [3]:
#Q 2.18 b
# 0.4674 - 1.984 * 0.03472
# 0.4674 + 1.984 * 0.03472
confint(model)

2.5 %      97.5 %   
(Intercept) -0.5309316 3.2619109
WingLength   0.3986288 0.5361792

We have 95% confident when sparrows's winglength increase 1 minimeter, the mean of weight will increase (0.399, 0.536) grams.

## Q 2.18 c 
CI is not 0, we rejected H0 when we did hypothesis test(part a), so we get Ha's conclusion. \
In part a, P-value is a small number. It still reject H0, the conclusion are the same.